File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/MedicinalProductUndesirableEffect.ts
import { CodeableConcept } from "./CodeableConcept";
import { Population } from "./Population";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";

export type MedicinalProductUndesirableEffect = DomainResource & {
    subject?: Reference[];
    population?: Population[];
    classification?: CodeableConcept;
    frequencyOfOccurrence?: CodeableConcept;
    symptomConditionEffect?: CodeableConcept;
};

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/MedicinalProductUndesirableEffect.py
from typing import Optional, List
from base import CodeableConcept
from base import Population
from base import DomainResource
from base import Reference

class MedicinalProductUndesirableEffect(DomainResource):
    subject: Optional[List[Reference]] = None
    population: Optional[List[Population]] = None
    classification: Optional[CodeableConcept] = None
    frequency_of_occurrence: Optional[CodeableConcept] = None
    symptom_condition_effect: Optional[CodeableConcept] = None